src/interface/variable.hpp

```diff
@@ -48,9 +48,11 @@ class CellVariable {
  public:
   /// Initialize a 6D variable
   CellVariable<T>(const std::string &label, const std::array<int, 6> dims,
-                  const Metadata &metadata)
+                  const Metadata &metadata, int sparse_id = -1)
```

Comment:
Can you make sparse_id const?

Reply:
For small data types, pass by value is preferable and `const` is not needed, because we get a copy.

Reply:
Yep, yep, your right. I saw your other comment, will need to make sure I remember that.